mne/coreg.py

```diff
@@ -39,6 +39,7 @@
 fid_fname = pformat(bem_fname, name='fiducials')
 fid_fname_general = os.path.join(bem_dirname, "{head}-fiducials.fif")
 src_fname = os.path.join(bem_dirname, '{subject}-{spacing}-src.fif')
+high_res_head_fname = os.path.join(subject_dirname, 'surf', 'lh.seghead')
```

Comment:
Typically you should if possible take the high res surface for
coregistration, if it is available. MNE analyze relies on users linking
them correctly ...

On Tue, Jul 5, 2016 at 5:08 PM Alexandre Gramfort <notifications@github.com>
wrote:

> In mne/coreg.py
> <https://github.com/mne-tools/mne-python/pull/3383#discussion_r69591426>:
>
> > @@ -39,6 +39,7 @@
> >  fid_fname = pformat(bem_fname, name='fiducials')
> >  fid_fname_general = os.path.join(bem_dirname, "{head}-fiducials.fif")
> >  src_fname = os.path.join(bem_dirname, '{subject}-{spacing}-src.fif')
> > +high_res_head_fname = os.path.join(subject_dirname, 'surf', 'lh.seghead')
>
> see what surface is produced by
>
> mne make_scalp_surfaces
>
> it's not the surface that should default to if you assume you ran "mne
> make_scalp_surfaces"
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/3383/files/f285ec969a9dc99a8fd3fe827184990a3b6ac382#r69591426>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AB0fitjdp-7vywfzwGX6TDZFyDUwSh4kks5qSoGBgaJpZM4JFP8_>
> .
>